src/Compilers/CSharp/Test/Emit/Emit/ResourceTests.cs

```diff
@@ -433,7 +433,7 @@ public void AddManagedResource()
             c1 = null;
         }
 
-        [ConditionalFact(typeof(DesktopOnly))]
+        [ConditionalFact(typeof(WindowsDesktopOnly))]
```

Comment:
>WindowsDesktopOnly [](start = 32, length = 18)

Is this change necessary?

Reply:
this is a cherry-pick from a PR I merged to master. see https://github.com/dotnet/roslyn/pull/30526#issuecomment-437505177

Reply:
I was getting the same transient failure in this branch.